plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -1077,6 +1083,25 @@ public void testNativeQueryIncorrectSyntax()
                 .failure().hasMessageContaining("Failed to get destination table for query");
     }
 
+    @Test
+    public void testQueriesWithParentProjectId()
+            throws Exception
+    {
+        DistributedQueryRunner queryRunner = BigQueryQueryRunner.builder()
+                .setConnectorProperties(ImmutableMap.<String, String>builder()
+                        .put("bigquery.arrow-serialization.enabled", "false")
```

Comment:
Why do we need to disable this property?

Reply:
It fails without it:
```
1) Error: BigQuery connector requires additional JVM arguments to run when 'bigquery.arrow-serialization.enabled' is enabled. Please add '--add-opens=java.base/java.nio=ALL-UNNAMED' to the JVM configuration.
  at BigQueryConnectorModule$ClientModule.verifyPackageAccessAllowed(BigQueryConnectorModule.java:154)
      \_ installed by: BigQueryConnectorModule -> BigQueryConnectorModule$ClientModule -> ConditionalModule -> BigQueryConnectorModule$ClientModule$$Lambda/0x00000fff00b4ba78
```